common/src/sql/dbinit.sql

```diff
@@ -1132,7 +1138,7 @@ CREATE TABLE omicron.public.instance_external_ip (
     ip_pool_range_id UUID NOT NULL,
 
     /* FK to the `project` table. */
```

Comment:
Since this is now nullable, I think it'd be worthwhile to add constraints on it like we have for `instance_id`. It should be pretty similar, but not identical, only allowing nulls here if this is a service IP address.

Reply:
Done - I added a constraint to the `project_id` specifically